src/components/orderTaking/PayScreen.re

```diff
@@ -10,6 +10,16 @@ type doing =
 type state = {
   doing,
   order: Order.orderVm,
+  orderItems: list(OrderItem.t),
```

Comment:
Why do you have all the order properties AND the order on the state in the pay screen? Isn't the pay screen only to update the payment status for an order?